npc/re/quests/quests_18_1.txt

```diff
@@ -0,0 +1,12429 @@
+			next;
+			mes "[Folklorist Gudra]";
+			mes "You seem to be interested in listening.";
+			mes "are you interested in this work?";
+			next;
+			switch( select( "I am interested", "I am not interested" ) ) {
+			case 1:
+				mes "[Folklorist Gudra]";
+				mes "If you are interested, does that mean you are willing to help?";
+				mes "Hands are always needed. I'd be happy to help.";
+				next;
+				mes "[Folklorist Gudra]";
+				mes "Can you write? Good.";
+				mes "Please record stories from the villagers.";
+				next;
+				mes "[Folklorist Gudra]";
+				mes "I have made an appointment to hear from a village child, a young man, and a grandmother.";
+				next;
+				mes "[Folklorist Gudra]";
+				mes "It's a long story, so you won't be able to hear it all in one day, so you need to help with a little bit of breathing.";
+				mes "Here, I'll give you a journal for recording.";
+				next;
+				mes "[Folklorist Gudra]";
+				mes "The child's name is Dina. The young man is a woman named Amira.";
+				mes "Then go to Shani or her grandmother, listen to the story, and write it down.";
+				setquest 16551;
+				setquest 16552;
+				setquest 16553;
+				getitem 1000408,1;	// Ep18_Recording_Note
+				close;
+			case 2:
+				end;
+			}
+			end;
+		}
+		if (countitem(1000408) < 1) {
+			getitem 1000408,1;	// Ep18_Recording_Note
+		}
+		mes "[Folklorist Gudra]";
+		mes "I have made an appointment to hear from a village child, a young man, and a grandmother.";
+		next;
+		mes "[Folklorist Gudra]";
+		mes "It's a long story, so you won't be able to hear it all in one day, so you need to help with a little bit of breathing.";
+		next;
+		mes "[Folklorist Gudra]";
+		mes "The child's name is Dina. The young man is a woman named Amira.";
+		mes "Then go to Shanina Grandma and listen to the story and write it down.";
+		close;
+	case 1:
+		mes "[Folklorist Gudra]";
+		mes "Dina and Amira, have you been writing stories to Grandma Shanina?";
+		mes "I'm excited about what kind of story it will be. Come on, show me your notebook.";
+		next;
+		mes "[Folklorist Gudra]";
+		mes "Uhhhhhh... This is a fairy tale that I can't tell my kids right now.";
+		mes "It contains discriminatory stereotypes that are unacceptable by today's standards.";
+		next;
+		mes "[Folklorist Gudra]";
+		mes "Oh, don't get me wrong. I just can't tell the kids, but it's a great resource for me.";
+		mes "Who did you hear this from?";
+		next;
+		mes "[Folklorist Gudra]";
+		mes "Are you saying this is the story that Shanina Grandma told me?";
+		mes "A story that her grandmother grew up listening to when she was young....";
+		next;
+		mes "[Folklorist Gudra]";
+		mes "I will read this carefully and organize it.";
+		mes "Please continue to receive stories from Shanina Grandma.";
+		next;
+		mes "[Folklorist Gudra]";
+		mes "I hope you will come back tomorrow and dictate the next part of this story.";
+		mes "Thank you. This is your pay for today's work.";
```

Comment:
![02](https://github.com/rathena/rathena/assets/147054311/7ccbfb88-6655-4ce8-a4c0-43325986f813)

Reply:
And all quest when finish can't close dialog.